x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/rule_creation/common_flows_supression_ess_basic.cy.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  ALERT_SUPPRESSION_FIELDS_INPUT,
+  ALERT_SUPPRESSION_FIELDS,
+  THRESHOLD_ENABLE_SUPPRESSION_CHECKBOX,
+  ALERT_SUPPRESSION_DURATION_INPUT,
+} from '../../../../screens/create_new_rule';
+
+import {
+  selectIndicatorMatchType,
+  selectNewTermsRuleType,
+  selectThresholdRuleType,
+} from '../../../../tasks/create_new_rule';
+import { startBasicLicense } from '../../../../tasks/api_calls/licensing';
+import { login } from '../../../../tasks/login';
+import { visit } from '../../../../tasks/navigation';
+import { CREATE_RULE_URL } from '../../../../urls/navigation';
+import { TOOLTIP } from '../../../../screens/common';
+
+import { deleteAlertsAndRules } from '../../../../tasks/api_calls/common';
+
+describe(
+  'Detection rules, Common flows Alert Suppression',
+  {
+    tags: ['@ess'],
+  },
+  () => {
+    describe('Create rule form', () => {
+      beforeEach(() => {
+        deleteAlertsAndRules();
+        login();
+        visit(CREATE_RULE_URL);
+        startBasicLicense();
+      });
+
+      it('can not create rule with rule execution suppression on basic license for all rules with enabled suppression', () => {
+        // Default query rule
+        cy.get(ALERT_SUPPRESSION_FIELDS_INPUT).should('be.disabled');
+        cy.get(ALERT_SUPPRESSION_FIELDS).trigger('mouseover');
```

Comment:
NIT: To wrap this into a meaningful task method and the other triggers as well.

Reply:
wrapped in a separate utility